src/CubedSpheres/cubed_sphere_set!.jl

```diff
@@ -42,3 +46,4 @@ function set_face_field!(field::ReducedField{LX, LY, Nothing}, f::Function) wher
 
     return nothing
 end
+=#
```

Comment:
Guessing we don't need this commented block of `set_face_field!` methods anymore?

Reply:
Nuked!